app/views/project_summary_mailer/project_summary.html.slim

```diff
@@ -0,0 +1,35 @@
+h3
+  = "Dear, #{@user.full_name}"
+p
+  | Here is a summary of the changed in your board
+p
+  - unless @user.new_tasks.empty?
+    |NEW TASKS 
+    - @user.new_tasks.each do |newtask|
+        = newtask.title
+        
+  - unless @user.deleted_tasks.empty?
+    |DELETED TASKS 
+    - @user.deleted_tasks.each do |deletetask|
+        = deletetask.title
+
+  - unless @user.completed_tasks.empty?
+    |COMPLETED TASKS 
+    - @user.completed_tasks.each do |completetask|
+        = completetask.title
+
+  - unless @user.unconfirmed_completed_tasks.empty?
+    |PENDING COMPLETED TASKS CONFIRMATIONS
+    - @user.unconfirmed_completed_tasks.each do |complete_confirm|
+        = complete_confirm.title
+
+  - unless @user.unconfirmed_deleted_tasks.empty?
+    |PENDING COMPLETED TASKS CONFIRMATIONS
+    - @user.unconfirmed_deleted_tasks.each do |delete_confirm|
+        = delete_confirm.title
+
+p
+  |Best regards,
+  br
+  i
+    | The "Learn IT, Girl" team
```

Comment:
@DMips In the ouput of the email, The first task appeared in the same line. How to bring it on next line.